chart/templates/_helpers.tpl

```diff
@@ -34,6 +34,7 @@ Create chart name and version as used by the chart label.
 Common labels
 */}}
 {{- define "joy-generator.labels" -}}
+tags.datadoghq.com/service: {{ .Values.datadogServiceLabel | default .Chart.Name }}
```

Comment:
```suggestion
tags.datadoghq.com/service: {{ .Values.datadogServiceLabel | default .Release.Name }}
```

What do you think about using the release name here instead?

Reply:
Sounds bueno!